team.md

```diff
@@ -86,6 +86,15 @@ Graduated from Queen's University at Kingston with an M.Sc. in physics. \
 Runs a radio show on CJLO. Likes local music.
 
 
+<img src="https://github.com/polyquantique/polyquantique.github.io/raw/master/assets/images/vincent.jpg"
+     alt="Vincent"
+     width="80"
+     height="90"
+     style="float: left; margin-right: 10px;" /> **Vincent Girouard**.  
+Undergraduate Student, [Engineering Physics](https://www.polymtl.ca/phys/), [Polytechnique Montréal](https://www.polymtl.ca).  
+2025 summer intern.
```

Comment:
Nice! You might want a little line about yourself... but only if you want. I will approve the PR anyway @Vigir2